internal/slacklog/generator.go

```diff
@@ -29,6 +29,10 @@ type HTMLGenerator struct {
 	baseURL string
 }
 
+// MAX_EMBEDDED_FILE_SIZE : 添付ファイルの埋め込みを行うファイルサイズ
+// これ以下の場合、表示されるようになる
+const MAX_EMBEDDED_FILE_SIZE = 100000
```

Comment:
Go の定数は camel case です。
また const にするなら外から変更できないし参照もしないので非公開が好ましいです。

```suggestion
const maxEmbeddedFileSize = 100000
```

なお外から変更させるなら `var` を使ってこうなります。

```suggestion
var MaxEmbeddedFileSize int64 = 100000
```

Reply:
camel case なのすっかり忘れてました…